packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -127,3 +127,28 @@ class ItemListNotifier<T> {
     _removedController.close();
   }
 }
+
+class SettingsFile {
+  SettingsFile.parse(String contents) {
+    for (String line in contents.split('\n')) {
+      line = line.trim();
+      if (line.startsWith('#') || line.isEmpty)
```

Comment:
Yup, they'll be removed. Right now this is just used to store our config information.